dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets

```diff
@@ -1108,31 +992,6 @@
 
   </Target>
 
-  <!-- Expand content of PRI files. -->
-  <Target Name="_ExpandPriUploadFiles"
-          Condition="'$(BuildAppxUploadPackageForUap)' == 'true'">
-
-    <ItemGroup>
-      <_PriUploadFilesToExpand Include="@(_PriFilesFromPayload)"
```

Comment:
>_PriUploadFilesToExpand [](http://example.com/codeflow?start=7&length=23)

no need to create upload file?

Reply:
yes. this isn't needed for us.

Reply:
if you search "upload" in the file, are other hits needed?